### PR TITLE
Add multisite explanation for Application Passwords.

### PR DESCRIPTION
**Pull request by georgestephanis**
Add explanation text in multisite environments when the user has multiple blogs clarifying that application passwords will grant access to all their sites.

Trac ticket: https://core.trac.wordpress.org/ticket/51489

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by TimothyBJacobs** [comment]
@georgestephanis looks like there are linting failures.

**Comment by georgestephanis** [comment]
Yeah, just fixed.

**Comment by TimothyBJacobs** [comment]
@SergeyBiryukov's also left comments on the ticket.

File: src/wp-admin/user-edit.php [hunk 1]
```diff
@@ -715,6 +715,29 @@
 	<div class="application-passwords hide-if-no-js" id="application-passwords-section">
 		<h2><?php _e( 'Application Passwords' ); ?></h2>
 		<p><?php _e( 'Application passwords allow authentication via non-interactive systems, such as XML-RPC or the REST API, without providing your actual password. Application passwords can be easily revoked. They cannot be used for traditional logins to your website.' ); ?></p>
+			<?php
+			if ( is_multisite() ) {
+				$blogs = get_blogs_of_user( $user->ID, true );
```
Comment (TimothyBJacobs) [inline src/wp-admin/user-edit.php:720]:
```suggestion
				$blogs = get_blogs_of_user( $user_id, true );
```

Fix variable name

**Pull request closed without merging**